package.json

```diff
@@ -1,8 +1,11 @@
 {
   "private": true,
   "version": "5.0.1",
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "boot": "./scripts/boot.sh",
+    "bootstrap": "git clean -dfx && yarn",
```

Comment:
Why not just "yarn"? Do we actually need to do git clean etc in here?

Wondering if we even now need any "bootstrap" script vs just `yarn`

Reply:
yes, we should probably remove this script

Reply:
The idea behind that is to have a "fresh" project. By running that command we reset the state of the  repository from the ground up (like a `git clone`). You can still run `yarn` since we deal with Workspaces the install will be correct. With the `bootstrap` the intent is a bit different.

Reply:
Ok got it. In which cases would you use bootstrap do you think? Since it can be dangerous to call git clean on the behalf of the user, I want to be sure we are providing the command for good reasons. Maybe you are already using it, let me know

Reply:
I use it when I create a build / run test to be sure to run those scripts in a clean environment (no install / build cache). The option provided remove **only** the files that are ignored in your configuration (`.gitignore`). Untracked files are untouched.